renderdoc/driver/ihv/amd/amd_counters.cpp

```diff
@@ -391,26 +445,89 @@ uint32_t AMDCounters::GetPassCount()
   return (uint32_t)numRequiredPasses;
 }
 
-uint32_t AMDCounters::BeginSession()
+uint32_t AMDCounters::CreateSession()
 {
-  gpa_uint32 sessionID = 0;
+  uint32_t sessionID = m_gpaSessionCounter;
+  GPA_SessionId gpaSessionId = nullptr;
```

Comment:
`NULL` rather than `nullptr` - there are a few other places in the commit, if you search/replace you should catch them all hopefully :smile:.

Reply:
Updated the changes. Thanks for the catch.